lib/lndclient/LndClient.ts

```diff
@@ -76,18 +82,27 @@ class LndClient extends BaseClient {
   }
 
   public connect = async () => {
-    if (!this.isDisabled()) {
+    if (this.isDisabled()) {
+      this.logger.error(`can't connect to lnd. lnd is disabled`);
+      return;
+    }
+    if (this.isConnected()) {
+      this.logger.error(`trying to connect to already connected lnd`);
```

Comment:
I think this should also be a `warn` and I'd rather it say "can't connect to lnd, lnd is already connected because "trying to connect" makes it sound like we're giong forward with the connection attempt.

Reply:
changed the text.

One should not try to connect to a connected LND. This should be considered an error.